clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/model/TitusInstance.groovy

```diff
@@ -102,6 +107,14 @@ class TitusInstance implements Instance {
     agentId
   }
 
+  String getIpv4Address() {
+    ipv4Address
+  }
+
+  String getIpv6Address() {
+    ipv6Address
```

Comment:
the earlier issue was due to this getter not being renamed with intellij refactor + groovy !